share/picongpu/tests/XrayScattering/include/picongpu/param/precision.param

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2013-2020 Rene Widera
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Define the precision of typically used floating point types in the
+ * simulation.
+ *
+ * PIConGPU normalizes input automatically, allowing to use single-precision by
+ * default for the core algorithms. Note that implementations of various
+ * algorithms (usually plugins or non-core components) might still decide to
+ * hard-code a different (mixed) precision for some critical operations.
+ */
+
+#pragma once
+
+
+namespace picongpu
+{
+
+/*! Select a precision for the simulation data
+ *  - precision32Bit : use 32Bit floating point numbers
+ *                     [significant digits 7 to 8]
+ *  - precision64Bit : use 64Bit floating point numbers
+ *                     [significant digits 15 to 16]
+ */
+#ifndef PARAM_PRECISION
+#   define PARAM_PRECISION precision32Bit
+#endif
+namespace precisionPIConGPU      = PARAM_PRECISION;
+
+/*! Select a precision special operations (can be different from simulation precision)
+ *  - precisionPIConGPU : use precision which is selected on top (precisionPIConGPU)
+ *  - precision32Bit    : use 32Bit floating point numbers
+ *  - precision64Bit    : use 64Bit floating point numbers
+ */
+namespace precisionSqrt          = precisionPIConGPU;
+namespace precisionExp           = precisionPIConGPU;
+namespace precisionTrigonometric = precisionPIConGPU;
+
+
+} // namespace picongpu
+
+#include "picongpu/unitless/precision.unitless"
```

Comment:
Is there a difference to the default case, if not please remove the file?

Reply:
```
#ifndef PARAM_PRECISION
#   define PARAM_PRECISION precision32Bit
#endif
namespace precisionPIConGPU      = PARAM_PRECISION;
```
I believe this part for setting the precision with flags is not included in the default file. @PrometheusPi